validator/submain.py

```diff
@@ -69,6 +69,9 @@ def timeout_handler(signum, frame):
             package = open(path, 'rb')
             test_package(err, package, path, expectation, for_appversions)
 
+        if err.detected_type == PACKAGE_EXTENSION:
+            err.metadata['is_extension'] = True
```

Comment:
I wonder if, given the name this should always be `False` or `True` but never not given. Although we do have a few other keys that behave this way I think. Still, can we make this explicitly set?